docker/Dockerfile

```
@@ -0,0 +1,37 @@
+FROM postgres:9.6
+
+ENV POSTGRES_VERSION 9.6
+ENV POSTGIS_VERSION 2.3
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        postgis \
+        postgresql-${POSTGRES_VERSION}-postgis-${POSTGIS_VERSION} \
+        postgresql-${POSTGRES_VERSION}-postgis-${POSTGIS_VERSION}-scripts
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        git \
+        ca-certificates \
+        build-essential \
+        autoconf \
+        automake \
+        zlib1g-dev \
+        postgresql-server-dev-all \
+        libxml2-dev \
+    && rm -rf /var/lib/apt/lists/* \
+    && git clone https://github.com/pgpointcloud/pointcloud \
+    && cd pointcloud \
+    && ./autogen.sh \
+    && ./configure CFLAGS="-Wall -Werror -O2 -g" \
```

Use the following code instead to fix this https://github.com/pgpointcloud/pointcloud/pull/182#issuecomment-396261740

```suggestion
    && ./configure --with-pgconfig=/usr/lib/postgresql/${POSTGRES_VERSION}/bin/pg_config CFLAGS="-Wall -Werror -O2 -g" \
```

See documentation: https://github.com/pgpointcloud/pointcloud#build